use super::GlobalError;

pub fn run() -> Result<(), GlobalError> {
    Ok(())
}

#[test]
fn challenge_day_7_one() -> Result<(),GlobalError> {
    assert_eq!(2,2);
    Ok(())
}

#[test]
fn challenge_day_7_two() -> Result<(),GlobalError> {
    assert_eq!(2,2);
    Ok(())
}